compose.yaml

```diff
@@ -0,0 +1,217 @@
+      - db_data:/bitnami/postgresql
+    ports:
+      - "127.0.0.1:5432:5432"
+
+  pgadmin:
+    image: dpage/pgadmin4:9.3.0
+    entrypoint: /bin/sh
+    command: >
+      -c "echo \"$$SERVERS_JSON\" > /tmp/servers.json &&
+      exec /entrypoint.sh"
+    environment:
+      SERVERS_JSON: |
+        {
+          "Servers": {
+            "1": {
+              "Name": "Mrija-Manager",
+              "Group": "Servers",
+              "Port": 5432,
+              "Username": "${POSTGRES_USER}",
+              "PasswordExecCommand": "echo '${POSTGRES_PASSWORD}'",
+              "Host": "db",
+              "SSLMode": "prefer",
+              "MaintenanceDB": "postgres"
+            }
+          }
+        }
+      PGADMIN_SERVER_JSON_FILE: /tmp/servers.json
+      PGADMIN_DEFAULT_EMAIL: 1@1.com
+      PGADMIN_DEFAULT_PASSWORD: "1"
+      PGADMIN_DISABLE_POSTFIX: "true"
+      PGADMIN_CONFIG_SERVER_MODE: "False"
+      PGADMIN_CONFIG_MASTER_PASSWORD_REQUIRED: "False"
+    depends_on:
+      db:
+        condition: service_healthy
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.pgadmin.rule=Host(`pgadmin.localhost`)"
+      - "traefik.http.routers.pgadmin.entrypoints=web"
+
+  # ors-app:
+  #   image: openrouteservice/openrouteservice:v9.0.0
+  #   ports:
+  #     - "8080:8082"
+  #     - "9001:9001"
+  #   user: "1000:1000" # Default is 1000:1000. make infra/up will override this to the actual user values
+  #   volumes:
+  #     - ./.docker/infra/ors:/home/ors
+  #   environment:
+  #     ors.engine.profile_default.build.source_file: /home/ors/files/sh.osm.pbf
+  #     REBUILD_GRAPHS: false
+  #     CONTAINER_LOG_LAVEL: INFO
+  #     XMS: 256m
+  #     XMX: 512m
+  #     ADDITIONAL_JAVA_OPTS: ""
+  #   deploy:
+  #     resources:
+  #       reservations:
+  #         cpus: "1"
+  #         memory: 256m
+  #       limits:
+  #         cpus: "1"
+  #         memory: 512m
+  #   networks:
+  #     - app-network
+
+  vroom:
+    image: ghcr.io/vroom-project/vroom-docker:v1.14.0
+    volumes:
+      - ./.docker/infra/vroom:/conf
+    environment:
+      VROOM_ROUTER: valhalla
+    deploy:
+      resources:
+        reservations:
+          cpus: "1"
+          memory: 50m
+        limits:
+          cpus: "1"
+          memory: 50m
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.vroom.rule=Host(`vroom.localhost`)"
+      - "traefik.http.routers.vroom.entrypoints=web"
+
+  valhalla:
+    image: ghcr.io/gis-ops/docker-valhalla/valhalla:3.5.1
+    volumes:
+      - ./.docker/infra/valhalla/custom_files:/custom_files
+    environment:
+      server_threads: 20
+    deploy:
+      resources:
+        reservations:
+          cpus: "1"
+          memory: 256m
+        limits:
+          cpus: "2"
+          memory: 1536m
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.valhalla.rule=Host(`valhalla.localhost`)"
+      - "traefik.http.routers.valhalla.entrypoints=web"
+
```

Comment:
maybe there could be an healthcheck on valhalla, because it does not have one itself. Valhalla has already one defined.